packages/components/src/ActionList/ActionList.scss

```diff
@@ -3,11 +3,14 @@
 /// @group Custom widgets
 ////
 
-$tc-action-list-item-width: 200px !default;
-$tc-action-list-item-icons-size: 20px !default;
-$tc-action-list-item-border-size: 2px !default;
+$tc-action-list-item-width: 20rem !default;
+$tc-action-list-item-icons-size: 2rem !default;
+$tc-action-list-item-border-size: 0.2rem !default;
 
 .tc-action-list {
+	width: fit-content;
```

Comment:
I know it will not be pleasant, but is it ok with IE11? I know that `fit-content` is not supported [https://caniuse.com/#search=fit-content](https://caniuse.com/#search=fit-content)
I try to check in it with Browserstack but Storybook was not ok with it.